docs/modules/ROOT/pages/documentation/ecosystem/vscode-extension/payara-server.adoc

```diff
@@ -52,6 +52,42 @@ Right-click on the application and select *Run on Payara Server* to deploy the a
 
 image::vscode-extension/payara-server/deploy-application.png[alt=Deploy application, width="60%"]
 
+Alternatively application can be deployed from *Java Projects* interface:
+
+image::vscode-extension/payara-server/java-projects-deploy.png[alt=Deploy application, width="60%"]
+
 The deploy application can be opened in the browser, undeployed or disabled by right-clicking on the application:
 
-image::vscode-extension/payara-server/list-application.png[alt=Open application homepage, width="60%"]
\ No newline at end of file
+image::vscode-extension/payara-server/list-application.png[alt=Open application homepage, width="60%"]
+
+[[deployment-settings]]
+== Deployment Settings
+
+The Auto Deploy and Hot Deploy mode are helpful for developers to run and test 
+an application immediately after making changes to its sources without 
+restarting the Server or manual redeployment to maximize your productivity.
+
+[[hot-deploy]]
+=== Hot Deploy
+
+_Since *5.201*&nbsp;_ Payara Server Community provides the Hot Deployment capability by boosting the performance 
+of the application redeployment operation and dynamically replaces the classloader of the application on each redeployment in the hot deploy mode.
+
+The Hot Deploy mode can be enabled in the Visual Studio Code from the *Deployment Settings* of the Payara Server.
+
+image::vscode-extension/payara-server/deployment-settings.png[Deployment Settings]
+image::vscode-extension/payara-server/deployment-settings-options.png[Enable Hot Deploy]
+
+NOTE: The Hot Deploy feature is in technical preview.
+
+After enabling the Hot Deploy settings as mentioned above, Simply deploy the Web Applications to Payara Server Community
+and modify any Java class of the application to perform the auto redeploy of application on save 
+and observer the fast redeployment of the application in Payara Server console log in compare to vanilla redeployment.
+
+
+[[auto-deploy]]
+=== Auto Deploy
+By enabling the Auto Deploy feature, you may automate the manual deployment 
+processes to boost the developers productivity. It will undeploy the existing 
+application and redeploy the new application on every save action so developers 
+can focus on the key development task.
```

Comment:
I would mention that this feature full redeploys an application and therefore is slower than hot redpeloy, which redeploys only the changed part of an application.